src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -43,13 +44,15 @@ const useGroupInfo = (channelId = null) => {
   const [isLoadingAddMember, setIsLoadingAddMember] = React.useState(false);
   const [uploadedImage, setUploadedImage] = React.useState('');
   const [isUploadingImage, setIsUploadingImage] = React.useState(false);
+  const [isUpdatingName, setIsUpdatingName] = React.useState(false);
   const [isLoadingInitChat, setIsLoadingInitChat] = React.useState(false);
   const [username, setUsername] = React.useState(channelState.channel?.data?.name);
   const [selectedUser, setSelectedUser] = React.useState(null);
   const [newParticipant, setNewParticipant] = React.useState([]);
   const [openModal, setOpenModal] = React.useState(false);
   const [isAnonymousModalOpen, setIsAnonymousModalOpen] = React.useState(false);
   const [isFetchingAllowAnonDM, setIsFetchingAllowAnonDM] = React.useState(false);
+  const [isOpenModalChangeName, setIsOpenModalChangeName] = React.useState(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New state variables have been introduced. Make sure these states are managed properly throughout the component lifecycle to avoid any inconsistencies.
```diff
+ const [isUpdatingName, setIsUpdatingName] = React.useState(false);
+ const [isOpenModalChangeName, setIsOpenModalChangeName] = React.useState(false);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->